requirements.txt

```diff
@@ -4,7 +4,7 @@ boto>=2.40.0
 apache-libcloud>=0.20.1
 click
 yamlordereddictloader
-ansible>=2.3.1
+ansible>=2.5.0
```

Comment:
Why are we updating the ansible version here? And if so, let's make sure to update the documentation.

Reply:
Ansible 2.5 implements `index_var` within the `loop_control` module.  We use it in the step of the playbook that creates disks if they do not exist

Reply:
Update: that code was changed and index_var is no longer necessary, so I removed it and reset the version